.github/workflows/apidocstest.yml

```
@@ -0,0 +1,94 @@
+name: apidocstests
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  test-api-docs:
+    runs-on: ubuntu-latest
```

We should update our other actions to always use a specific version

```suggestion
    runs-on: ubuntu-16.04
```